consensus/fork.go

```diff
@@ -98,6 +98,7 @@ func (s *State) generateAndApplyDiff(bn *blockNode) (err error) {
 	s.currentBlockID = bn.block.ID()
 	s.currentPath[bn.height] = bn.block.ID()
 
+	// TODO: minerSubsidy isn't used anywhere?
```

Comment:
woah what?

I have testing for this, the miner fees do appear in the block subsidy. Unless I screwed that up.

Oh wait you might not have those tests in this branch.